_pages/status/Helmi-SW-update.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Helmi Software Update'
+date: 2023-10-07
+permalink: /_pages/status/Helmi-SW-update/
+header:
+  teaser: /assets/images/about-icon.jpg
+published: true
+layout: single
+tags:
+  - helmi
+  - maintenance
+---
+
+*Helmi will be undergoing a software update. Here is a list of changes*
+
+# Software Package changes
+
+The IQM adapter packages will be updated. 
+
+| Package        | Previous version | New Version   |
+|----------------|------------------|---------------|
+| iqm-client     | 2.2              | >=12.5 < 13.0 |
+| iqm-cortex-cli | -                | >=3.1 < 4.0   |
+| qiskit-iqm     | 2.0              | >=8.3 < 9.0   |
+| cirq-iqm       | 4.1              | >=11.9 < 12.0 |
+
+As a result you will be able to use `qiskit~=0.42.1` and `cirq~=1.1.0` which provide some significant changes. 
```

Comment:
Should we list some of the changes here? 

e.g. heralding

Reply:
Fake Adonis available. batched job submission and parameterized circuits for qiskit? These are not available in cirq: https://iqm-finland.github.io/qiskit-on-iqm/user_guide.html#more-advanced-examples

Reply:
I could add them with a different heading or just add them to the Software Package changes while specifying that it is only available for qiskit